### DISCUSSION TITLE
How to obtain the blob URL for a filtered view?

### DISCUSSION THREAD

**Discussion by aborruso** [githubocto/flat-action#36] [Q&A] [answered]
Hi,
when I apply a filter, I can download the filtered file and the URL is something like:

```
blob:https://flatgithub.com/0374b99f-8a49-4539-a112-585352d977c7
```

Is there some API call, is there some other way, to obtain programmatically the download URL that corresponds to the URL I used to load the flat viewer page?

Thank you

**Comment by Wattenberger** [comment, marked as answer]
Unfortunately, the data is downloaded through the flat-ui component (https://github.com/githubocto/flat-ui/blob/main/src/components/grid.tsx#L240)

An alternative could be to add a postprocessing step to your Flat action that saves a filtered version of your data.

**Reply by aborruso** [reply]
I am sorry that it is not possible, because otherwise it would add another piece to this fantastic project: the API (sort of).

You will have something like https://gitrows.com/
